elements/rh-code-block/rh-code-block.css

```diff
@@ -127,6 +127,28 @@
   color: inherit;
 }
 
+#content::slotted(:is(code[class*='language-'], pre[class*='language-'])) {
```

Comment:
Not sure this will work in all cases. Prism usually outputs a code wrapped in a pre, hence the need for lightdom css.

Reply:
It was the only way we could figure out how to pivot the styles without reflecting "wrap" back to the host element as the only switch is on the shadow container.  Otherwise we could have modified `prism.css.ts` with a `rh-code-block[wrap]` style selector and style it on this layer.

Reply:
@bennypowers not sure how to effectively test for the outliers or when they would pop up, if you have an gumption to take a stab.